src/period-types/PeriodTypes.component.jsx

```diff
@@ -0,0 +1,215 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { CenteredContent, CircularLoader } from '@dhis2/ui'
+import { getInstance as getD2 } from 'd2'
+import CheckboxMaterial from 'material-ui/Checkbox'
+import React, { useState, useCallback } from 'react'
+import settingsActions from '../settingsActions.js'
+import styles from './PeriodTypes.module.css'
+
+const query = {
+    periodTypes: {
+        resource: 'periodTypes',
+    },
+    dataOutputPeriodTypes: {
+        resource: 'configuration/dataOutputPeriodTypes',
+    },
+}
+
+const formatPeriodTypeName = (name) => {
+    const simpleLabels = {
+        Daily: i18n.t('Daily'),
+        Weekly: i18n.t('Weekly'),
+        Monthly: i18n.t('Monthly'),
+        BiMonthly: i18n.t('Bi-monthly'),
+        Yearly: i18n.t('Yearly'),
+        BiWeekly: i18n.t('Bi-weekly'),
+        Quarterly: i18n.t('Quarterly'),
+        SixMonthly: i18n.t('Six-monthly'),
+    }
+
+    if (simpleLabels[name]) {
+        return simpleLabels[name]
+    }
+
+    const monthMap = {
+        April: 'April',
+        July: 'July',
+        Oct: 'October',
+        Nov: 'November',
+    }
+
+    if (name.startsWith('Weekly')) {
+        const day = name.replace('Weekly', '')
+        return day
```

Comment:
the day won't be translated here, so you'd need to also create a dayMap and add translations for the weekdays